src/day8/re_frame/trace.cljs

```diff
@@ -176,10 +178,12 @@
 
 (defn render-trace-panel []
   (let [filter-input     (r/atom "")
-        filter-items     (r/atom [])
+        filter-items     ((fnil #(r/atom %) [])
```

Comment:
I felt like `[]` is too specific of a default value. In the end I rewrote the function like this:

```clojure
(let [filter-items (r/atom (or (localstorage/load "filter-items")
                               []))])
```